sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py

```diff
@@ -403,6 +406,8 @@ def update_message(
         :type message_id: str
         :param content: Chat message content.
         :type content: str
+        :param metadata: Message metadata.
```

Comment:
This will need to be documented as keyword rather than param:
`:keyword dict[str, str] metadata : Message metadata.`